### ISSUE TITLE
fxtract hits headers that are the exact reverse of query

### ISSUE THREAD

**Issue by geronimp** [ctSkennerton/fxtract#8]
Hey, 

I ran into the following issue, simplified here:

```
$ fxtract -c -H -X 1660334 reads.fa
2
```

where reads.fa contains:

```
>1660334
CTAGTGATTAGAGTTTGATCCTGGCTCAGGATGAACGCTGGCGGCGTGCTTAACACATGCAAGTCGAACGGAAAGGCCCAAGCTTGCTTGGGTACTCGAGTGGCGAACG
>4330661
GTCACTGCTATTGGGATTCGACTAAGCCATGCGAGTCGAGAGGGTTCGGCCCTCGGCTGACGGCCCAGTAACACGTGGATAATCTACCCTAAGGCGGAGGATAATCTCG
```

I only want the seq with the header 1660334. Notice the second hit, 4330661, is the exact inverse of 1660334.

Am I missing a flag that stops this from occuring?

Thanks

**Comment by ctSkennerton** [comment]
Ah, this is left over from an old "feature" of fxtract where searching for a sequence would also search for its reverse complement. I didn't think that it still did this. Obviously a bug, I'll fix it soon as there is no work around currently

**Comment by ctSkennerton** [comment]
Should be fixed with new version 2.1

**Comment by geronimp** [comment]
Thanks!
